From e35181a09e3bde2021e1dd7542cf4622653cba11 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Wed, 9 May 2012 12:27:19 -0700
Subject: [PATCH 094/795] MR1: CLOUDERA-BUILD. TestWebUIAuthorization.testWebUIAuthorizationForCommonServlets is failing

  Reason: testcases failures after HADOOP-8343 integration
  Author: Alejandro Abdelnur
  Ref: CDH-5691
(cherry picked from commit 476e30a90b1fe0d3fbb80a70205d2009a5c77108)

(cherry picked from commit eee48181abf754b518cac6ccb4fb7c9cd427ccdc)
---
 .../hadoop/mapred/TestWebUIAuthorization.java      |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/src/test/org/apache/hadoop/mapred/TestWebUIAuthorization.java b/src/test/org/apache/hadoop/mapred/TestWebUIAuthorization.java
index 013b3e3..785297c 100644
--- a/src/test/org/apache/hadoop/mapred/TestWebUIAuthorization.java
+++ b/src/test/org/apache/hadoop/mapred/TestWebUIAuthorization.java
@@ -754,7 +754,8 @@ public void testWebUIAuthorizationForCommonServlets() throws Exception {
     Properties props = new Properties();
     props.setProperty("hadoop.http.filter.initializers",
         DummyFilterInitializer.class.getName());
-    props.setProperty(CommonConfigurationKeys.HADOOP_SECURITY_AUTHORIZATION, "true");
+      props.setProperty(CommonConfigurationKeys.HADOOP_SECURITY_AUTHORIZATION, "true");
+      props.setProperty(CommonConfigurationKeys.HADOOP_SECURITY_INSTRUMENTATION_REQUIRES_ADMIN, "true");
     props.setProperty(JobConf.MR_ADMINS, mrAdminUser + " " + mrAdminGroup);
 
     startCluster(true, props);
-- 
1.7.0.4

